src/Android/WebAuthCallbackActivity.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Android.App;
+using Android.Content.PM;
+
+namespace Bit.Droid
+{
+    [Activity(NoHistory = true, LaunchMode = LaunchMode.SingleTop)]
+    [IntentFilter(new[] { Android.Content.Intent.ActionView },
+        Categories = new[] { Android.Content.Intent.CategoryDefault, Android.Content.Intent.CategoryBrowsable },
+        DataScheme = "bitwarden")]
+    public class WebAuthCallbackActivity : Xamarin.Essentials.WebAuthenticatorCallbackActivity { }
```

Comment:
Should we register this with an explicit name? See other activities for example:


    [Register("com.x8bit.bitwarden.WebAuthCallbackActivity")]

If not, I think it creates some kind of random activity name in the manifest, which might change over time and break things.

We had this problem with the autofill service changing names and getting disabled from update to update, so I have always tried to be explicit since then.

Reply:
I don't believe so, since the WebAuthenticator intent originates from within the app which can include the (known) activity name as an extra.  With autofill the intent originates from the system, which has no knowledge of the receiver's activity name.

Reply:
Understood